GLMakie/src/rendering.jl

```diff
@@ -31,7 +31,12 @@ function render_frame(screen::Screen; resize_buffers=true)
     ShaderAbstractions.switch_context!(nw)
 
     function sortby(x)
-        return x[3][:model][][3, 4]
+        robj = x[3]
+        plot = screen.cache2plot[robj.id]
```

Comment:
The plot lookup was the expensive bit here, so would be nice if we could avoid it!

Reply:
Dict lookups are O(1) and somewhere around 10ns. My benchmarks for the current solution only have 3% difference to the old one.
Using `robj[:model]` is also what made this pr fail, because lines with linestyles apply it on the CPU so it does not end up in uniforms. I would also expect this to subtly fail with f32 converts because those can change the model matrix (set it to I after applying it on the CPU)